packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2180,6 +2180,8 @@ export const DEFAULT_COMPONENT_ICONS: {
     CaptionsOffIcon: JSX.Element;
     CaptionsSettingsIcon: JSX.Element;
     ChangeSpokenLanguageIcon: JSX.Element;
+    ContextMenuCameraIcon: JSX.Element;
```

Comment:
Can you make this more specific, something like `ControlButtonContextMenuCameraIcon` to indicate where this icon is used?